docs/reference/src/main/asciidoc/gettingstarted.asciidoc

```diff
@@ -33,12 +33,24 @@ You can also deploy the generated artifact (war or ear) to any other
 container that supports Java EE 7, such as GlassFish 4.
 
 The sections below cover the steps for deploying with Maven in detail.
-Let's start with WildFly.
+
+=== First try
+
+If you simply want to run the example without the requirement of a specific runtime then start with the following commands.
```

Comment:
Well, it should work for both examples. But you're right, it will be better to be more specific.